server/src/main/java/com/ibm/ws/lars/rest/FrontPage.java

```diff
@@ -44,20 +55,28 @@
 
     @Override
     protected void doGet(HttpServletRequest req, HttpServletResponse resp) throws ServletException, IOException {
-        resp.setContentType(MediaType.TEXT_HTML);
+        resp.setContentType(MediaType.APPLICATION_JSON);
         resp.setCharacterEncoding(StandardCharsets.UTF_8.name());
 
-        int count = serviceLayer.countAllAssets(Collections.<AssetFilter> emptyList(), null);
+        List<AssetFilter> filters = new ArrayList<>();
+        filters.add(new AssetFilter("state", Arrays.asList(new Condition[] { new Condition(Operation.EQUALS, "published") })));
+        int assetCount = serviceLayer.countAllAssets(filters, null);
+
+        JsonObjectBuilder frontPageJsonBuilder = getJsonObjectBuilder();
+        frontPageJsonBuilder.add("serverName", "LARS");
+        frontPageJsonBuilder.add("assetCount", assetCount);
+        JsonObject frontPageJsonObject = frontPageJsonBuilder.build();
 
         PrintWriter out = resp.getWriter();
-        out.println("<!DOCTYPE html>");
-        out.println("<html>");
-        out.println("<head><title>LARS is running</title></head>");
-        out.println("<body>");
-        out.println("<h1>LARS is running</h1>");
-        out.println("<p>The repository is running with " + count + " assets</p>");
-        out.println("</body>");
-        out.println("</html>");
+        out.print(frontPageJsonObject);
```

Comment:
It might make more sense here to use a JsonWriter rather than relying on the PrintWriter and the JsonObject.toString() result.